onnx_tf/frontend.py

```diff
@@ -496,7 +496,8 @@ def _pool_op(cls, node, onnx_op, **kwargs):
         onnx_op, [node.inputs[0]], [node.name],
         pads=pads,
         kernel_shape=kernel_shape,
-        strides=strides)
+        strides=strides,
+        **kwargs.get("external_node_kwargs", {}))
```

Comment:
This is a bit sketchy... The problem with using kwargs is that we won't be able to catch unexpected keyword arguments error... I'd suggest that we do it the conventional way by having another argument with default value, is there a reason against this approach?